src/extensions/common/NiceTable.tsx

```diff
@@ -14,9 +15,9 @@ interface Col {
 interface Props {
     rows: Row[],
     columns: Col[],
-    onDeleteRow?: ((row: Row) => void),
+    onDeleteRow?: ((key: string, columnValues: {[key: string]: any}) => void),
```

Comment:
The types passed into `onDeleteRow` and `onEditRow` look to match the `Row` interface defined above, and it looks like that had been structured that way previously--was this a conscious decision to not use that interface in this version, or did something get mixed up in combining the files?

Reply:
Nice catch, yeah the columnValues should not be there. I'll put this in a new issue.